IPython/core/tests/test_alias.py

```diff
@@ -38,4 +38,25 @@ def test_alias_args_error():
     with capture_output() as cap:
         _ip.run_cell('parts 1')
 
-    nt.assert_equal(cap.stderr.split(':')[0], 'UsageError')
\ No newline at end of file
+    nt.assert_equal(cap.stderr.split(':')[0], 'UsageError')
+    
+def test_alias_args_commented():
+    """Check that alias correctly ignores 'commented out' args"""
+    _ip.magic('alias commetarg echo this is %%s a commented out arg')
+    
+    with capture_output() as cap:
+        _ip.run_cell('commetarg')
```

Comment:
Did you ment `comme**n**targ` here and above ? (actually we don't care as it is a test you could have called it `rfgtrederfgghtui` ...